common/metrics.py

```
@@ -0,0 +1,209 @@
+            Hit rate as a percentage (0.0 to 100.0), or 0.0 if no data.
+        """
+        try:
+            hits = cls.cached_queryset_hit.labels(model=model)._value._value
+            misses = cls.cached_queryset_miss.labels(model=model)._value._value
+            total = hits + misses
+            
+            if total == 0:
+                return 0.0
+                
+            return (hits / total) * 100.0
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)